.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
 
     steps:
       - name: Set up PHP
-        uses: shivammathur/setup-php@2.3.1
```

Comment:
This is the only change Im not sure about.

Reply:
They seem to have been reliably moving the v2 tag, just like the checkout code action.

![image](https://user-images.githubusercontent.com/2829600/85921118-9cd15080-b871-11ea-8385-366724d6348e.png)